public/dashboard.css

```diff
@@ -38,123 +48,149 @@ div.stat-box {
 }
 
 span.stat {
-  font-size: 5vh;
+  font-family: 'Roboto Mono', monospace;
+  font-size: 4rem;
 }
 
 span.stat-label {
-  font-size: 2vh;
+  font-size: 1.25rem;
 }
 
 div.events {
-  flex: 3;
+  grid-row: 5 / 7;
+  grid-column: 1;
+  border: 16px solid #616161;
   display: flex;
   flex-direction: column;
   justify-content: space-around;
-  background-color: #ffc0c0;
   overflow: hidden;
 }
 
-span.event {
-  padding: 0.5vh;
-  font-size: 3vh;
+div.events > * {
+  font-size: 2em;
+  padding: calc(var(--dashboard-radius)*2);
 }
 
 div.notices {
-  flex: 4;
-  background-color: #ffffc0;
-  padding: 0.5vh;
-  font-size: 3vh;
+  grid-row: 1 / 4;
+  grid-column: 1;
+  border: 16px solid #ffd54f;
   overflow: hidden;
 }
 
+div.notices > * {
+  font-size: 2em;
+  margin: calc(var(--dashboard-radius)*4) calc(var(--dashboard-radius)*2);
+}
+
 div.puzzles {
-  flex: 1;
-  display: flex;
-  flex-direction: column;
-  background-color: #c0c0ff;
-  overflow: hidden;
+  grid-row: 1 / 7;
+  grid-column: 2;
+  border: 16px solid #1976d2;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(10, 1fr);
+  grid-gap: var(--dashboard-radius);
 }
 
 div.puzzle {
-  width: 100%;
+  font-size: 40px;
+  display: grid;
+  grid-gap: calc(var(--dashboard-radius) * 2);
+  grid-template-columns: 64px 2fr min-content;
+  width: calc(100% - (4 * var(--dashboard-radius)));
+  margin: 0 auto;
+  padding: var(--dashboard-radius);
+  justify-self: center;
+  align-self: center;
+  align-items: center;
+  border: 2px solid #e1e1e1;
+}
+
+div.pinned {
```

Comment:
this pinned class is not used yet? do we need some code support to make this possible, or can we figure out which puzzles to apply this style to via CSS selectors (e.g. searching the div.puzzle descendant elements for an element with class pin or something)?

Reply:
although maybe now this is fixed, i was reviewing before you updated